File: 1678.py
class Solution:
    def interpret(self, command: str) -> str:
        res = ""
        index = 0
        while(index < len(command)):
            val = command[index]
            if(val == 'G'):
                res += 'G'
            elif(val == '('):
                if(command[index + 1] == ')'):
                    res += 'o'
                    index += 1
                else:
                    res += 'al'
                    index += 3
            index += 1
        return res

File: 1678.cpp
class Solution {
public:
    string interpret(string command) {
        string res = "";
        for(int i = 0; i < command.size(); i++) {
            if(command[i] == '(') {
                if(command[i + 1] == ')') {
                    res += 'o';
                    i++;
                }
                else {
                    res += "al";
                    i += 3;
                }
            } else {
                res += 'G';
            }
        }
        return res;
    }
};